views/event_details.hbs

```diff
@@ -1,22 +1,21 @@
-<div class="eventDetailsHeader">
-    {{>header}}
-  <div class="prevBtn ">
-    <a href="{{{back}}}"><span class="glyphicon glyphicon-chevron-left"></span></a>
-  </div>
-</div>
+{{>header}}
 
-<div class="event-details text-center">
-  <h1 class="text-warning eventName">{{event.en.name}}</h1>
-  {{#if imageUrl}}
-  <img src="{{event.imageUrl}}" alt="event picture">
+<div class="event-details">
+  {{#if event.imageUrl}}
+    <img class="event-img center" src="{{event.imageUrl}}" alt="event picture">
```

Comment:
So instead of an `img`, make this a `div` and set the background-image here (also get rid of the center class since it's redundant):

`<div class="event-img" style="background-image: url('{{event.imageUrl}}')"></div>`

Then give the `event-img` class these properties:
```css
height: 20rem;
background-size: cover;
background-position: center;
box-shadow: 0 .2rem 1rem #bbb;
```

Reply:
👍